Source/WebCore/dom/Element.cpp

```diff
@@ -461,13 +461,12 @@ static ShouldIgnoreMouseEvent dispatchPointerEventIfNeeded(Element& element, con
 #else
         UNUSED_PARAM(platformEvent);
 #endif
-        if (platformEvent.syntheticClickType() != SyntheticClickType::NoTap)
-            return ShouldIgnoreMouseEvent::No;
```

Comment:
I'm guessing this is safe to do now that the click event is a PointerEvent? But would we still not face the redundant event issue we tried to address in https://commits.webkit.org/218757@main?

Reply:
Maybe @whsieh remembers?

Reply:
Right — please double check that a single tap over any element with a `click` event handler doesn't emit redundant pointerdown/pointerup events.

Reply:
I guess I can just leave this here and just add a check for click/auxclick/contextmenu instead, to be safe